lib/bolognese/utils.rb

```diff
@@ -959,17 +967,16 @@ def from_schema_org_contributors(element)
     end
 
     def map_hash_keys(element: nil, mapping: nil)
-      Array.wrap(element).map do |a|
-        a.map {|k, v| [mapping.fetch(k, k), v] }.reduce({}) do |hsh, (k, v)|
-          if v.is_a?(Hash)
-            hsh[k] = to_schema_org(v)
-            hsh
-          else
-            hsh[k] = v
-            hsh
-          end
+      element = element.to_h if element.is_a?(Hash)
```

Comment:
This would probably improve with some better naming and description @richardhallett